action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -234,14 +254,20 @@ def _parse_permissions(transaction: dict, **kwargs) -> Optional[dict]:
     if not isinstance(action_ids, list):
         print(f"Function {function} came up with {action_ids} which is not a valid list.")
         return
+    to = transaction['to']
+    to_name = addr.reversebook.get(to, "!!NOT-FOUND")
+    if to_name == "20210418-authorizer/Authorizer":
+        to_string = "Authorizer"
+    elif isinstance(to_name, str):
+        to_string = f"!!f{to_name}??"
```

Comment:
Siggestion to ,ake it `NOT_FOUND` or something

Reply:
In this case it is found, and has a name, it's just not the correct one. If it is really NOT-FOUND it is already set to NOT-FOUND by line 258.